lib/Form/Processor/ContentFormProcessor.php

```diff
@@ -165,4 +173,16 @@ private function saveDraft(ContentStruct $data, $languageCode)
 
         return $contentDraft;
     }
+
+    /**
+     * @param ContentUpdateData|ContentCreateData|NewnessChecker $data
+     *
+     * @return string
+     */
+    private function resolveMainLanguageCode($data): string
+    {
+        return $data->isNew()
```

Comment:
Should we be a tiny bit protective and test the type of `$data`, since we don't have typehinting here ?

Reply:
Good idea. Fixed.